units/host_monitoring_idle_termination.go

```diff
@@ -205,7 +209,7 @@ func (j *idleHostJob) getTimesFromHost(ctx context.Context, h *host.Host) (time.
 			"message":  "host termination for a non-ephemeral distro",
 			"cause":    "programmer error",
 		})
-		return 0, 0, errors.New("attempted to terminate non-ephemeral host")
+		return true, fmt.Errorf("attempted to terminate non-ephemeral host '%s'", h.Id)
```

Comment:
should use errors.Errorf; this also returns the stack trace in case we ever need it